charts/kubernetes-agent/templates/nfs-pod-pvc.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "nfs.podPvcName" . }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+{{- if not (quote .Values.persistence.nfs.podStorageClassName | empty) }}
+  storageClassName: {{quote .Values.persistence.nfs.podStorageClassName }}
+{{- end }}
```

Comment:
this allows them to use storageClassName="" and have it work?

Reply:
Yep, I've tested this and this:
```
helm template \
--set persistence.nfs.podStorageClassName="" \
--set tentacle.logLevel="Trace" \
--set tentacle.ACCEPT_EULA="Y" \
--set tentacle.targetName="scme" \
--set tentacle.serverUrl="http://host.docker.internal:8066/" \
--set tentacle.serverCommsAddress="http://host.docker.internal:10943/" \
--set tentacle.space="Default" \
--set tentacle.targetEnvironments="{dev}" \
--set tentacle.targetRoles="{scme-role}" \
--set tentacle.serverApiKey="API-LRJCU6XSMGQQCLC2W5UQQOZGCH6Z06T" \
--create-namespace --namespace octopus-agent-scme \
scme \
/Users/scottmerchant/Documents/dev/helm-charts/charts/kubernetes-agent
```
comes back including this:
```
# Source: kubernetes-agent/templates/nfs-pod-pvc.yml
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: octopus-agent-nfs-scme-pod-pvc
spec:
  accessModes:
    - ReadWriteOnce
  storageClassName: ""
  resources:
    requests:
      storage: 1Gi
```